app/models/journeys/further_education_payments/provider/session_answers.rb

```diff
@@ -22,6 +47,29 @@ def claim
         def dfe_sign_in_service_access?
           !!dfe_sign_in_service_access
         end
+
+        def claim_verified?
+          verification.present?
+        end
+
+        # We need to do this to get the base form to set existing verification
+        # attributes from the session on the verification form.
+        attribute :assertions_attributes, pii: false
+        def assertions_attributes
+          verification
+            .fetch("assertions", [])
+            .map(&:with_indifferent_access)
+            .index_by(&:itself)
+        end
+
+        def identity_verification_required?
+          unless FeatureFlag.enabled?(:fe_provider_identity_verification)
```

Comment:
unfortunately not on master yet but somewhere I added `disabled?` to avoid double negations